Formula/zebra.rb

```diff
@@ -22,6 +22,12 @@ class Zebra < Formula
   depends_on "icu4c"
   depends_on "yaz"
 
+  uses_from_macos "bzip2"
+  uses_from_macos "expat"
+  uses_from_macos "libxml2"
+  uses_from_macos "libxslt"
```

Comment:
Still failing so will need to probably try locally to check logs:
```
  checking for module dom... disabled
  configure: error: Cannot enable mod-dom because of missing libs (XML, etc)
```

Reply:
Need to rebuild `yaz` with `libxslt` support.